File: C++/Example Array/Change Data (Array).cpp
#include <iostream>
#include <string>

using namespace std;

int main()
{

	char varlttr[10] = {'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j'};
	varlttr[1] = 'x';
	varlttr[9] = 'z';
	
	cout << "Letter : " << varlttr[1] << endl;
	cout << "Letter : " << varlttr[9] << endl;

}

File: C#/Example Array/Change Data (Array).cs
using System;

namespace Main
{
    class main
    {
        static void Main(string[] args)
        {
            char[] varlttr = { 'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j' };
            varlttr[1] = 'x';
            varlttr[9] = 'z';

            Console.WriteLine($"Letter : {varlttr[1]}");
            Console.WriteLine($"Letter : {varlttr[9]}");
        }
    }
}
